internal/terraform/exec/exec.go

```diff
@@ -29,6 +29,10 @@ var passthroughEnvVars = []string{
 	// os.TempDir would otherwise fall back to C:\Windows on Windows
 	// which has no write permissions for non-admins
 	"TMPDIR", "TMP", "TEMP",
+	// This allows backends and providers to authenticate
+	// using external auth provider and/or proxy
+	"PATH", "path", "PATHEXT",
```

Comment:
I'm unsure if this is case-sensitive, I'm assuming it depends on the OS, but I'm just following Go's own `os.LookPath` implementation here, which looks at `$path` (lowercase) in case of Windows.